app/controllers/cost_of_living_landing_page_controller.rb

```diff
@@ -1,7 +1,11 @@
 class CostOfLivingLandingPageController < ApplicationController
   slimmer_template "gem_layout_full_width"
 
+  rescue_from GdsApi::ContentStore::ItemNotFound, with: :show
```

Comment:
Can we add a further comment in the commit message that we will remove this when we publish the special router?